### PR TITLE
[workspace] update packages descriptions and README headlines

### PR DESCRIPTION
**Pull request by Simek**
# Why

While working on RND updates I have spotted that not all packages have a correct/nice description of the functionality.

# How

Update the several packages description fields and README files headlines to match each other a bit more closer, expand the tags lists, make sure that content more-or-less follows our writing guide. 

# Test Plan

N/A

### CODE REVIEW

File: packages/expo-navigation-bar/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "expo-navigation-bar",
   "version": "3.0.4",
-  "description": "Interact with the system navigation bar",
+  "description": "Modify and observe the native navigation bar on Android devices",
```
Comment (behenate) [inline packages/expo-navigation-bar/package.json:4]:
```suggestion
  "description": "Modify and observe the native navigation bar on Android devices.",
```

File: packages/expo-screen-capture/package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "expo-screen-capture",
   "version": "6.0.1",
-  "description": "ExpoScreenCapture standalone module",
+  "description": "Protect screens in your app from being captured or recorded, and notifies if a screenshot is taken.",
```
Comment (behenate) [inline packages/expo-screen-capture/package.json:4]:
```suggestion
  "description": "Protects screens in your app from being captured or recorded, and notifies if a screenshot is taken.",
```

File: packages/expo-symbols/package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "expo-symbols",
   "version": "0.1.4",
-  "description": "Expo Symbols",
+  "description": "Provides access to the SF Symbols library on iOS for React Native and Expo apps",
```
Comment (behenate) [inline packages/expo-symbols/package.json:4]:
```suggestion
  "description": "Provides access to the SF Symbols library on iOS for React Native and Expo apps.",
```

**Review by behenate — APPROVED** [review]
Left a few suggestions 😄

**Review by douglowder — APPROVED** [review]

File: packages/expo-clipboard/README.md [hunk 4]
```diff
@@ -7,7 +7,7 @@
   </a>
 </p>
 
-`expo-clipboard` provides an interface for getting and setting Clipboard content on Android, iOS, and Web.
```
Comment (brentvatne) [inline packages/expo-clipboard/README.md:10]: we use oxford comma, so the comma here is correct. https://www.notion.so/expo/Writing-Guide-35a1eb87298a49c2ad80f8a4f21b7c52#69050271bd0f499e81ea432ab1b42c77

File: packages/expo-notifications/package.json [hunk 5]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "expo-notifications",
   "version": "0.28.3",
-  "description": "Notifications module",
+  "description": "Provides an API to fetch push notification tokens and to present, schedule, receive and respond to notifications.",
```
Comment (brentvatne) [inline packages/expo-notifications/package.json:4]:
```suggestion
  "description": "Provides an API to fetch push notification tokens and to present, schedule, receive, and respond to notifications.",
```

**Review by brentvatne — APPROVED** [review]
thanks for cleaning these up!!

File: packages/expo-clipboard/README.md [hunk 6]
```diff
@@ -7,7 +7,7 @@
...
+`expo-clipboard` provides an interface for getting and setting Clipboard content on Android, iOS and Web.
```
Comment (Simek) [inline packages/expo-clipboard/README.md:10]:
```suggestion
`expo-clipboard` provides an interface for getting and setting Clipboard content on Android, iOS, and Web.
```

**Pull request merged**